core/templates/dev/head/base.html

```diff
@@ -225,10 +225,44 @@
                             </ul>
                           </li>
                         {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if-start="windowWidth > 768"></create-exploration-button>
+                          {% if SHOW_CUSTOM_PAGES %}
+                            <li class="oppia-clickable-navbar-element pull-right">
+                              <a class="oppia-navbar-tabs" href="/forum">
+                                Forum
+                              </a>
+                            </li>
+                          {% endif %}
+                          <li class="dropdown oppia-clickable-dropdown pull-right">
+                            <a class="oppia-navbar-tabs">
+                              About
+                              <span class="caret"></span>
+                            </a>
+                            <ul class="dropdown-menu oppia-navbar-profile-dropdown" ng-mouseover="onMouseoverDropdownMenu($event)" ng-mouseleave="onMouseoutDropdownMenu($event)">
```

Comment:
The oppia-navbar-profile-dropdown class seems misnamed. If that class contains CSS specific to the profile dropdown, a new one should be created. Otherwise, perhaps that class should be renamed to .oppia-navbar-dropdown or similar.